frontend/src/scenes/funnels/FunnelBarGraph.tsx

```diff
@@ -323,7 +323,6 @@ export function FunnelBarGraph({ filters, dashboardItemId, color = 'white' }: Om
                             <div className="funnel-bar-wrapper">
                                 {Array.isArray(step.nested_breakdown) && step.nested_breakdown?.length ? (
                                     step.nested_breakdown.map((breakdown, index) => {
-                                        const conversionRate = calcPercentage(breakdown.count, basisStep.count)
```

Comment:
Not sure about this change, aren't we breaking the conversion calculation when using breakdowns?